docs/DeveloperGuide.md

```diff
@@ -555,6 +553,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   
   Use case ends.
     
+**Use case: UC05 - Undoing a command**
+
+**MSS**
+
+1. User executes a command.
```

Comment:
Would it be better to specify what is the exact command? So that number 2 can also be more specific what was being undone

Reply:
I get what you mean, but at the same time I think the MSS should be general. I guess I can put the example in brackets?

Reply:
Oh thats true. Ok its fine to leave it as such then if u hvnt made the changes